src/react-utils/useNodeHovered.ts

```diff
@@ -6,6 +6,7 @@ export const useNodeHovered = (nodeRef: RefObject<HTMLElement>) => {
     const [nodeHovered, setNodeHovered, unsetNodeHovered] = useBooleanState(false);
 
     useEffect(() => {
+        // TODO: @makhnatkin is used current?
```

Comment:
@d3m1d0v Should we change the dependencies from nodeRef to nodeRef.current for the warning fix if there is no ESLint: React House Effect has a missing dependency: 'nodeRef'. Either include it or remove the dependency array. Mytable values like 'nodeRef.current' aren't valid dependencies because mutating them doesn't re-render the component.(react-hooks/exhaustive-deps) ?